### ISSUE TITLE
User Suggestion: Webp conversion

### ISSUE THREAD

**Issue by ov3qPI** [mylifeandcode/BulkImageConverter#3]
.webp has become a common file type and supporting its bulk conversion would be very useful.

Cheers.

**Comment by mylifeandcode** [comment]
Agreed. Would also like to add support for HEIC. It's just a matter of finding the time. This is a side project of mine, so the time I allot to it is limited. Thanks again for the suggestion, and I'll try to make this a part of the app.
